messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/CommonAttributes.java

```diff
@@ -105,7 +105,7 @@
             .build();
 
     SimpleAttributeDefinition BRIDGE_CONFIRMATION_WINDOW_SIZE = create("confirmation-window-size", INT)
-            .setDefaultValue(new ModelNode(FileConfiguration.DEFAULT_CONFIRMATION_WINDOW_SIZE))
+            .setDefaultValue(new ModelNode(ActiveMQDefaultConfiguration.getDefaultBridgeConfirmationWindowSize()))
```

Comment:
Our default values need to be statically declared in our own code base, and not be dependent on external libraries which can change without us noticing.

That or we have to have some test that will notice if any of the values change so we can update the API version, do transformation etc.

The code you're changing already had this problem, so you're not introducing any new problem.